### PR TITLE
Update UserCallingSettings/Delegates for GA

### PR DESCRIPTION
**Pull request by JensTrier**
Remove Public preview note
Be more specific on Identity support
Add reference to Teams PS Module 4.0.0
Added note to Get/Set/Remove-CsVoiceNormalizationRule that the cmdlet has been deprecated from Teams PS Module 4.0.0

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: teams/teams-ps/teams/Get-CsUserCallingSettings.md [hunk 1]
```diff
@@ -15,8 +15,6 @@ schema: 2.0.0
 ## SYNOPSIS
 This cmdlet will show the call forwarding, simultaneous ringing, call group and delegation settings for a user.
 
-> [!NOTE]
-> **Preview** The use of this cmdlet is in Public Preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsUserCallingSettings.md:18]:
```suggestion
 
```

File: teams/teams-ps/teams/Remove-CsUserCallingDelegate.md [hunk 2]
```diff
@@ -15,8 +15,6 @@ schema: 2.0.0
 ## SYNOPSIS
 This cmdlet will remove a delegate for calling in Microsoft Teams.
 
-> [!NOTE]
-> **Preview** The use of this cmdlet is in Public Preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsUserCallingDelegate.md:18]:
```suggestion
```

File: teams/teams-ps/teams/Set-CsUserCallingSettings.md [hunk 3]
```diff
@@ -15,8 +15,6 @@ schema: 2.0.0
 ## SYNOPSIS
 This cmdlet will set the call forwarding, simultaneous ringing and call group settings for the specified user.
 
-> [!NOTE]
-> **Preview** The use of this cmdlet is in Public Preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsUserCallingSettings.md:18]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested changes to remove the extra blank lines. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested changes to remove the extra blank lines. Many thanks.

Thanks & Done

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**